_includes/misc/spellchecker-languages.md

```diff
@@ -0,0 +1,29 @@
+## Supported languages
+
+The following languages are supported for the Spell Checker Pro plugin. All of the listed languages are supported for customers using {{site.cloudname}} {{site.productname}} version 5.9 or higher.
```

Comment:
This reads very strangely... it says the `using Tiny Cloud TinyMCE version 5.9 or higher`. Maybe this instead?
```suggestion
The following languages are supported for the Spell Checker Pro plugin. All of the listed languages are supported for customers using {{site.productname}} version 5.9 or higher on {{site.cloudname}}.
```